components.tsx

```diff
@@ -148,57 +148,58 @@ interface PostPageProps {
 export function PostPage({ post, state }: PostPageProps) {
   const html = gfm.render(post.markdown);
   return (
-    <div class="max-w-screen-sm px-6 pt-8 mx-auto">
-      <div class="pb-16">
-        <a
-          href="/"
-          class="inline-flex items-center gap-1 text-sm text-gray-500/80 hover:text-gray-700 transition-colors"
-          title="Back to Index Page"
-        >
-          <svg
-            className="inline-block w-5 h-5"
-            viewBox="0 0 20 20"
-            xmlns="http://www.w3.org/2000/svg"
+    <Fragment>
+      {state.showHeaderOnPostPage && state.header}
```

Comment:
disclaimer, you may see lots of changes here when reviewing, but the only actual change are these 2 lines  as well as the closing </Fragment> :trollface: 

whatever below is just indentation

Reply:
deno fmt?

Reply:
sorry, you mean I should run deno fmt here? But I already run 🤔 and there is nothing to fmt already

![image](https://user-images.githubusercontent.com/28719400/177675378-68a01734-a241-497e-8158-76aa6d02dc53.png)